composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/var-dumper": "^3.1.10"
+        "symfony/var-dumper": "~4.0"
```

Comment:
How about using the `^4.0` and I think it can be better that using `~4.0` version :+1:.